catalog/core/catalog-core-impl/filter-proxy/src/main/java/ddf/catalog/filter/proxy/adapter/GeotoolsFilterAdapterImpl.java

```diff
@@ -319,6 +325,27 @@ public Object visit(PropertyIsEqualTo filter, Object delegate) {
         String functionName = filterValues.functionName;
         List<Object> functionArgs = filterValues.functionArgs;
 
+        //Special case to handle relative temporal queries
+        if (literal instanceof String && RELATIVE_TEMPORAL_MATCHER.matcher((String) literal)
+                .matches()) {
+            DateTime currentDateTime = new DateTime();
+
+            //Split out the ISO 8601 duration from the ECQL function
+            String duration = (((String) literal).split("[\\(\\)]"))[1];
```

Comment:
If the literal doesn't have this pattern, this could be an ArrayIndexOutOfBoundsException.  Also, we may want to consider precompiling this pattern like we do for the `RELATIVE_TEMPORAL_MATCHER`